cmd/podman/shared/create.go

```diff
@@ -699,61 +623,86 @@ func ParseCreateOpts(ctx context.Context, c *GenericCLIResults, runtime *libpod.
 		pidsLimit = 0
 	}
 
+	pid := cc.PidConfig{
+		PidMode: pidMode,
+	}
+	ipc := cc.IpcConfig{
+		IpcMode: ipcMode,
+	}
+
+	cgroup := cc.CgroupConfig{
+		Cgroups:      c.String("cgroups"),
+		Cgroupns:     c.String("cgroupns"),
```

Comment:
I'd prefer to put all namespace configuration together if we can

Reply:
Even if we do i'd still like proceessing of them to be separated so if a pod only shares a couple of namespaces, all fields don't have to be configured fully